api/base/serializers.py

```diff
@@ -835,14 +835,13 @@ class JSONAPIRelationshipSerializer(ser.Serializer):
     Provides a simplified serialization of the relationship, allowing for simple update request
     bodies.
     """
-    id = ser.CharField(source='node._id', required=False, allow_null=True)
+    id = ser.CharField()
```

Comment:
It makes sense to not define `source` in the base class, but I'm not sure that `required` and `allow_null` should have been removed here.